dspace-server-webapp/src/test/java/org/dspace/app/rest/WorkspaceItemRestRepositoryIT.java

```diff
@@ -5181,23 +5181,21 @@ public void deleteWorkspaceItemWithMinRelationshipsTest() throws Exception {
         parentCommunity = CommunityBuilder.createCommunity(context)
                                           .withName("Parent Community")
                                           .build();
-        Collection col1 = CollectionBuilder
-            .createCollection(context, parentCommunity).withName("Collection 1").build();
+        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity).withName("Collection 1")
```

Comment:
Although it's good to test the new scenario, it's not ok that the current setup is no longer tested

Can you restore the original tests, and add these as separate new tests instead?